make/CreateJmods.gmk

```diff
@@ -81,13 +81,11 @@ endif
 ifneq ($(CMDS_DIR), )
   DEPS += $(call FindFiles, $(CMDS_DIR))
   ifeq ($(call isTargetOs, windows)+$(SHIP_DEBUG_SYMBOLS), true+public)
```

Comment:
```suggestion
  ifeq ($(call isCompiler, microsoft)+$(SHIP_DEBUG_SYMBOLS), true+public)
```

stripped pdbs (and pdbs in general) are only ever used with the VS toolchain

Reply:
I understand the point of this change, but it is not directly with the issue addressed here (i.e. this condition wasn't introduced in this PR.).
Should it be included in the PR anyway?

Reply:
On second thought, hold on for a while

Reply:
I agree with @fthevenet, such a change should be separate from this.

Reply:
I'll retract my suggestion then, and include this change after this is integrated on my own